### PR TITLE
Smaller Updates to Handel

### PR DESCRIPTION
**Pull request by nibhar**
While doing some code review with @hrxi we found a couple of minor changes we wanted to make. None of these should affect how the code executes with the exception of the early return which should just make code which executed unnecessarily no longer execute.

### CODE REVIEW

File: handel/src/store.rs [hunk 1]
```diff
@@ -230,20 +230,21 @@ where
                 .get_mut(level)
                 .unwrap_or_else(|| panic!("Missing Level {level}"))
                 .combine(&identity, false);
+            // ggf expect() here
```
Comment (hrxi) [inline handel/src/store.rs]: Leftover comment.

File: handel/src/store.rs [hunk 2]
```diff
@@ -230,20 +230,21 @@ where
...
 
             self.individual_contributions
                 .get_mut(level)
                 .unwrap_or_else(|| panic!("Missing Level {level}"))
                 .insert(identity, contribution.clone());
+            // ggf expect ggf check for change
```
Comment (hrxi) [inline handel/src/store.rs]: Leftover comment.

**Review by Eligioo — APPROVED** [review]

File: handel/src/store.rs [hunk 3]
```diff
@@ -159,7 +165,10 @@ impl<TId: Clone + std::fmt::Debug + 'static, TProtocol: Protocol<TId>>
                 contributors
             }
         } else {
-            registry.signers_identity(&contribution.contributors())
+            // Combining worked meaning the contributions did not overlap. Since individuals should already
+            // have been present in the previous best this the best signature available.
```
Comment (jsdanielh) [inline handel/src/store.rs]:
```suggestion
            // have been present in the previous best this is the best signature available.
```

Comment (hrxi) [reply handel/src/store.rs]: One more word to the left.

**Review by ii-cruz — APPROVED** [review]
LGTM

**Pull request merged**